cellbender/remove_background/data/dataset.py

```diff
@@ -1062,6 +1078,105 @@ def get_matrix_from_cellranger_h5(filename: str) \
             'barcodes': barcodes}
 
 
+def get_matrix_from_anndata(filename: str) \
+        -> Dict[str, Union[sp.csr.csr_matrix, List[np.ndarray], np.ndarray]]:
+    """Load a count matrix from an h5ad AnnData file.
+
+    The file needs to contain raw counts for all measured barcodes in the 
+    `.X` attribute or a `.layer[{'counts', 'spliced'}]` attribute.  This function
+    returns a dictionary that includes the count matrix, the gene names (which
+    correspond to columns of the count matrix), and the barcodes (which
+    correspond to rows of the count matrix).
+
+    This function works for any AnnData object meeting the above requirements,
+    as generated by alignment methods like `kallisto | bustools`.
+
+    Args:
+        filename: string path to .h5ad file that contains the raw gene
+            barcode matrices
+
+    Returns:
+        out['matrix']: scipy.sparse.csr.csr_matrix of unique UMI counts, with
+            barcodes as rows and genes as columns
+        out['barcodes']: numpy array of strings which are the nucleotide
+            sequences of the barcodes that correspond to the rows in
+            the out['matrix']
+        out['gene_names']: List of numpy arrays, where the number of elements
+            in the list is the number of genomes in the dataset.  Each numpy
+            array contains the string names of genes in the genome, which
+            correspond to the columns in the out['matrix'].
+        out['gene_ids']: List of numpy arrays, where the number of elements
+             in the list is the number of genomes in the dataset.  Each numpy
+             array contains the string Ensembl ID of genes in the genome, which
+             also correspond to the columns in the out['matrix'].
+        out['feature_types']: List of numpy arrays, where the number of elements
+             in the list is the number of genomes in the dataset.  Each numpy
+             array contains the string feature types of genes (or possibly
+             antibody capture reads), which also correspond to the columns
+             in the out['matrix'].
+
+    """
+    logging.info(f"Detected AnnData format")
+    
+    adata = anndata.read_h5ad(filename)
+    
+    if "counts" in adata.layers.keys():
+        # this is a common manual setting for users of scVI
+        # given the manual convention, we prefer this matrix to 
+        # .X since it is less likely to represent something other
+        # than counts
+        logging.info("Found `.layers['counts']. Using for count data.")
+        count_matrix = adata.layers["counts"]
+    elif "spliced" in adata.layers.keys():
+        # alignment using kallisto | bustools with intronic counts
+        # does not populate `.X` by default, but does populate
+        # `.layers['spliced'], .layers['unspliced']`.
+        # we use spliced counts for analysis
+        logging.info("Found `.layers['spliced']. Using for count data.")
```

Comment:
Same here, I would add another backtick after ".layers['spliced']" to enclose that part in backticks
```
logging.info("Found `.layers['spliced']`. Using for count data.")
```